packages/instrumenter/src/mutators/node-mutator.ts

```diff
@@ -1,6 +1,10 @@
 import type { types, NodePath } from '@babel/core';
 
-export interface NodeMutator {
-  mutate(path: NodePath): Iterable<types.Node>;
+import { NodeMutatorConfiguration, MutationLevel } from '../mutation-level/mutation-level.js';
+
+export interface NodeMutator<T extends keyof MutationLevel> {
+  mutate(path: NodePath, levelMutations: string[] | undefined): Iterable<types.Node>;
```

Comment:
Instead of having each `NodeMutator` decide on whether to generate a mutant, it would be cleaner if they always generate everything and let excluding be done afterward (in the `BabelInstrumenter` class). It would safe you a lot if-statements and features like "absolute mutant score" would be easier to implement later.